docs/developer-docs/latest/development/plugins-extension.md

```diff
@@ -16,24 +16,33 @@ Plugin extensions code is located in the `./src/extensions` folder (see [project
 ```bash
 /extensions
   /some-plugin-to-extend
-    strapi-admin.js
     strapi-server.js
     /content-types
       /some-content-type-to-extend
         model.json
       /another-content-type-to-extend
         model.json
   /another-plugin-to-extend
-    strapi-admin.js
+    strapi-server.js
 ```
 
 :::
 
 Plugins can be extended in 2 ways:
 
-- [extending the plugin's Content-Types](#extending-a-plugin-s-content-types)
+- [extending the plugin's content-types](#extending-a-plugin-s-content-types)
 - [extending the plugin's interface](#extending-a-plugin-s-interface) (e.g. to add controllers, services, policies, middlewares and more)
 
+::: note
```

Comment:
forking or using patch-package. patch-package is generally the recommended method.